code/game/objects/effects/decals/remains.dm

```diff
@@ -27,12 +27,26 @@
 	desc = "They look like the remains of a small reptile."
 	icon_state = "lizard"
 
-/obj/effect/decal/remains/attack_hand(mob/user as mob)
-	to_chat(user, "<span class='notice'>[src] sinks together into a pile of ash.</span>")
+//Target turns to ash or oil.
+/obj/effect/decal/remains/proc/crumble()
 	var/turf/simulated/floor/F = get_turf(src)
-	if (istype(F))
-		new /obj/effect/decal/cleanable/ash(F)
+	if (icon_state == "remainsrobot")
+		visible_message(SPAN_NOTICE("[src] degrades into a pool of oil."))
+		if (istype(F))
+			new /obj/effect/decal/cleanable/blood/oil(F)
+	else
+		visible_message(SPAN_NOTICE("[src] sinks together into a pile of ash."))
```

Comment:
```suggestion
		visible_message(SPAN_NOTICE("\The [src] sink together into a pile of ash."))
```
And do this for the robot ones, too.
What \The does is it adds it in front of improper nouns, but leaves it off of proper nouns (ie: Anything capitalized). since `name = "remains"` (lowercase r) it will add "The" in front of it. "The remains sink ..." flows better than "remains sinks."